### PR TITLE
Use MediaPlayer

### LINKED ISSUE #2758: Secret recovery phrase video not playing on iOS devices
**Describe the bug**

Secret recovery phrase video not playing/displaying for iOS devices 

**Screenshots**

IMG_7E38B2707F15-1

**To Reproduce**
_Steps to reproduce the behavior_
1. Launch app in fresh state
2. Create a wallet
3. On Secure your wallet view, notice video isn't displayed

If you already have a wallet then do the following:
- burger menu
- settings
- security & privacy

IMG_445293F8D774-1

**Expected behavior**

Video should be displayed and playable 

**Smartphone (please complete the following information):**
 - Device: iPhones
 - App Version 2.5.0

-------------------------------------------------------------
_to be added after bug submission by internal support / PM_
**Severity**

- How critical is the impact of this bug on a user?
- Add stats if available on % of customers impacted
- Is this visible to all users?
- Is this tech debt?

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

Resolves #2758

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

**Issue**

Resolves #2758

### CODE REVIEW

**Review by ibrahimtaveras00 — APPROVED** [review]
Works now on iPhone devices, QA Passed 👍🏽

File: app/components/UI/SeedPhraseVideo/index.js [hunk 1]
```diff
@@ -1,63 +1,29 @@
...
 import PropTypes from 'prop-types';
 import { StyleSheet, View } from 'react-native';
-import AndroidMediaPlayer from '../../Views/MediaPlayer/AndroidMediaPlayer';
-import Video from 'react-native-video';
-import Device from '../../../util/Device';
-import Loader from '../../Views/MediaPlayer/Loader';
+import MediaPlayer from '../../Views/MediaPlayer';
 import scaling from '../../../util/scaling';
 
+const height = scaling.scale(240);
```
Comment (andrepimenta) [inline app/components/UI/SeedPhraseVideo/index.js:7]: I would make this all uppercase to indicate it's a constant, wdyt?

**Review by andrepimenta — APPROVED** [review]
Just made a small comment, other than that LGTM! 👍

File: app/components/UI/SeedPhraseVideo/index.js (hunk 1 above)
Comment (rickycodes) [reply app/components/UI/SeedPhraseVideo/index.js]: done!

**Pull request merged**